hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -2800,6 +2800,17 @@
       available_inout_list: []
       param_list:
       [
+        {
+          name: MemInitFile
+          desc: Hexfile to initialize the OTP macro.
+          type: ""
+          default: '''""'''
```

Comment:
:smile:

Reply:
yeah I guess I can remove some of the quotes here and it'll still work (since we don't need multiline)...

Reply:
ah this is generated, never mind then :)